syntax = "proto3";

package MvdS.ksvc;

message CryptoKeyVersion {
  string name = 1;
};
